setup.py

```diff
@@ -69,6 +69,42 @@
     ],
     package_data = {'nltk': ['test/*.doctest', 'VERSION']},
     install_requires = ['six>=1.10.0'],
+    extras_require = {
```

Comment:
1. It seems that these package versions have been specified almost 3 years ago. So I think we can safely remove the minimum version requirements if there are no objections.

2. I like your solution. I was trying to have something like this importing a separate module, but - as you said in the other comment - this was getting overly complex without any good reason.

Thank you!